server/graphql/v2/mutation/CreateCollectiveMutation.js

```diff
@@ -32,6 +35,13 @@ async function createCollective(_, args, req) {
     host = await fetchAccountWithReference(args.host, { loaders });
   }
 
+  if (remoteUser) {
+    const rateLimit = new RateLimit(`collective_create_${remoteUser.id}`, 60, ONE_HOUR_IN_SECONDS, true);
+    if (!(await rateLimit.registerCall())) {
+      throw new RateLimitExceeded();
+    }
+  }
+
```

Comment:
@Betree added rate limiting there

Reply:
No rate limit when unauthenticated? Maybe we can do something like:

```es6
const rateLimitKey = remoteUser ? `collective_create_${remoteUser.id}` : `collective_create_ip_${req.ip}`
```